cdap-test/src/main/java/co/cask/cdap/test/TestManager.java

```diff
@@ -399,7 +389,7 @@ ArtifactManager addPluginArtifact(ArtifactId artifactId, Set<ArtifactRange> pare
    * @deprecated since 3.4.0. Use {@link ArtifactManager#delete()} instead
    */
   @Deprecated
-  void deleteArtifact(Id.Artifact artifactId) throws Exception;
+  void deleteArtifact(ArtifactId artifactId) throws Exception;
```

Comment:
don't change this. It is already a deprecated method. We can remove the deprecated methods in this class in another pass, once we make sure none of the downstream projects are using them.